File: src/assets/usage-examples-code/windows/refresh_window/refresh_window-1-basic-usage.cpp
#include "splashkit.h"

int main()
{
    window wind = open_window("Refresh Window Example", 800, 600);

    write_line("Refreshing the window...");

    clear_window(wind, COLOR_WHITE);
    draw_text("Window will be refreshed.", COLOR_BLACK, 50, 50);
    refresh_window(wind);

    write_line("Window refreshed.");

    delay(3000); // Delay to keep the window open for 3 seconds

    return 0;
}

File: src/assets/usage-examples-code/windows/refresh_window/refresh_window-1-basic-usage.cs
using System;
using SplashKitSDK;

Window wind = SplashKit.OpenWindow("Refresh Window Example", 800, 600);

Console.WriteLine("Refreshing the window...");

SplashKit.ClearWindow(wind, SplashKit.ColorWhite());
SplashKit.DrawText("Window will be refreshed.", SplashKit.ColorBlack(), 50, 50);
SplashKit.RefreshWindow(wind);

Console.WriteLine("Window refreshed.");

SplashKit.Delay(3000); // Delay to keep the window open for 3 seconds
